R/render.r

```diff
@@ -36,7 +36,7 @@ render_page <- function(pkg = ".", name, data, path = "", depth = NULL, quiet =
 
   # The real location of 404.html is dynamic (#1129).
   # Relative root does not work, use the full URL if available.
-  if(identical(path, "404.html") && length(pkg$meta$url)){
+  if(path == "404.html" && length(pkg$meta$url)){
```

Comment:
After you've merged this PR can you please fix the style here (and below)? Should be a space and after the ()`.